atomic_reactor/plugins/post_tag_and_push.py

```diff
@@ -148,86 +148,82 @@ def run(self):
-                    if self.need_skopeo_push() or source_docker_archive:
-                        self.push_with_skopeo(registry_image, insecure, docker_push_secret,
-                                              source_docker_archive)
-                    else:
-                        # OSBS2 TBD either use store manifest from ManifestUtil
-                        # or tag_imag from utils.image
-                        # we won't need pushing
-                        # self.tasker.tag_and_push_image(self.workflow.data.image_id,
-                        #                                registry_image, insecure=insecure,
-                        #                                force=True, dockercfg=docker_push_secret)
-                        pass
-
-                    if source_docker_archive:
-                        manifests_dict = get_all_manifests(registry_image, registry, insecure,
-                                                           docker_push_secret, versions=('v2',))
-                        try:
-                            koji_source_manifest_response = manifests_dict['v2']
-                        except KeyError as exc:
-                            raise RuntimeError(
-                                f'Unable to fetch v2 schema 2 digest for {registry_image.to_str()}'
-                            ) from exc
-
-                        wf_data.koji_source_manifest = koji_source_manifest_response.json()
-
-                    digests = get_manifest_digests(registry_image, registry,
-                                                   insecure, docker_push_secret)
-
-                    if (not (digests.v2 or digests.oci) and (retry < max_retries)):
-                        sleep_time = DOCKER_PUSH_BACKOFF_FACTOR * (2 ** retry)
-                        self.log.info("Retrying push because V2 schema 2 or "
-                                      "OCI manifest not found in %is", sleep_time)
-
-                        time.sleep(sleep_time)
-                    else:
-                        break
-
-                pushed_images.append(registry_image)
-
-                tag = registry_image.to_str(registry=False)
-                push_conf_registry.digests[tag] = digests
-
-                if not config_manifest_digest and (digests.v2 or digests.oci):
-                    if digests.v2:
-                        config_manifest_digest = digests.v2
-                        config_manifest_type = 'v2'
-                    else:
-                        config_manifest_digest = digests.oci
-                        config_manifest_type = 'oci'
-                    config_registry_image = registry_image
+        insecure = self.registry.get('insecure', False)
+
+        docker_push_secret = self.registry.get('secret', None)
+        self.log.info("Registry %s secret %s", self.registry['uri'], docker_push_secret)
+
+        for image in wf_data.tag_conf.images:
+            if image.registry:
```

Comment:
pre_tag_from_config is already adding registry

Reply:
removed the check and raise